test/sql/version.sql

```diff
@@ -0,0 +1,3 @@
+-- Just test that the function does not error out as the output
+-- will change.
+select count(*) from _timescaledb_internal.get_git_version();
```

Comment:
I don't like the name get_get_version. Sounds like getting the version of git. maybe get_timescaledb_git() or get_git_commit()